eo-maven-plugin/src/test/java/org/eolang/maven/CleanMojoTest.java

```diff
@@ -92,9 +91,7 @@ void fullCompilingLifecycleSuccessfully(@TempDir final Path temp) throws IOExcep
             .with("ignoreTransitive", true)
             .with(
                 "objectionary",
-                (Objectionary) input -> new InputOf(
-                    "[] > sprintf\n"
-                )
+                new OyFake(s -> new InputOf("[] > sprintf\n"))
```

Comment:
@MikhailLipanin You are using that statement many times. Maybe it's better to create default constructor for it?

Reply:
@volodya-lombrozo, there are a lot of similar, but not the same cases. For example `OyFake` with object `sprintf` used only 3 times:
![image](https://user-images.githubusercontent.com/95661085/208769422-c9af26f0-6489-49a7-b51a-6333cf6a031a.png)

The point is that in some cases, there is important, what the object is inside `OyFake`, so we can't just replace all of the occurrences of `OyFake` for the default `s -> new InputOf("[] > sprintf\n")` .

So, I think there is no any default constructor for `OyFake`, WDYT?

Reply:
@MikhailLipanin Do we really need to use different objects in all that cases?

Reply:
@volodya-lombrozo, Yes, since in tests like `OyFallbackSwapTest`, we `assert` that the right object is being saved

Reply:
done